node/modules/lp2p/pubsub.go

```diff
@@ -114,6 +114,22 @@ func GossipSub(in GossipIn) (service *pubsub.PubSub, err error) {
 		InvalidMessageDeliveriesDecay:  pubsub.ScoreParameterDecay(time.Hour),
 	}
 
+	ingestTopicParams := &pubsub.TopicScoreParams{
```

Comment:
please dont just copy paste these params, you may weaken security properties of the other topics.

Is there proper validation on this topic?
I dont see anything. If not, dont set parameters at all, they are worse than useless.

Reply:
@vyzo Do you mean that `topicParams[build.IngestTopic(in.Nn)]` should not be set?  Or, are you saying that it should be set to some default, empty `TopicScoreParams` or `nil`?
```
topicParams[build.IngestTopic(in.Nn)] = &pubsub.TopicScoreParams{}
```

Reply:
unless you have a strong validator (which i dont see anywhere), do *not* set topic parameters.